cumulusci/tasks/debug_logs.py

```diff
@@ -0,0 +1,135 @@
+import datetime
+import os
+import csv
+
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+
+class TurnOnDebugLogs(BaseSalesforceApiTask):
+    task_docs = """
+    Turn on debugging information in Salesforce
+    """
+
+    debug_level_id = None  # Class variable
+
+    def _run_task(self):
+        """Create a TraceFlag for a given user."""
+        self._delete_debug_levels()
+        self._delete_trace_flags()
+        self.logger.info("Creating DebugLevel object")
+        DebugLevel = self._get_tooling_object("DebugLevel")
+        result = DebugLevel.create(
+            {
+                "ApexCode": "Info",
+                "ApexProfiling": "Debug",
+                "Callout": "Info",
+                "Database": "Info",
+                "DeveloperName": "CumulusCI",
+                "MasterLabel": "CumulusCI",
+                "System": "Info",
+                "Validation": "Info",
+                "Visualforce": "Info",
+                "Workflow": "Info",
+            }
+        )
+        if TurnOnDebugLogs.debug_level_id:
+            self.logger.info(
+                "Debug logs are already turned on ({TurnOnDebugLogs.debug_level_id})"
+            )
+            return
```

Comment:
I don't quite understand what this is trying to do. If the debug level is already set up, shouldn't we return up at the top of this method before deleting and recreating it?

But a class attribute is not a great place to keep track of that since it'll be lost between multiple runs of cci or multiple users running cci on different machines. I think querying the DebugLevel object in the org would be the thing to do.